addons/mail/static/src/models/device/device.js

```diff
@@ -12,25 +13,23 @@ registerModel({
             this._onResize = _.debounce(() => this._refresh(), 100);
         },
         _willDelete() {
-            window.removeEventListener('resize', this._onResize);
+            browser.removeEventListener('resize', this._onResize);
```

Comment:
Can't use messaging here? It's already deleted?
This is kind of a flaw in our approach then.